Misc/NEWS.d/next/Library/2021-11-28-22-44-54.bpo-45917.lNn_gz.rst

```diff
@@ -0,0 +1 @@
+Added a function that returns 2 raised to a given number :func:`math.exp2`
```

Comment:
Should this file be deleted? (It looks as though we've ended up with two separate news entries.)

Reply:
👍